src/Renderer/Pdf.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * Class for rendering the barcode in PDF resource
+ *
+ * @deprecated since 2.8.0; to be removed in 3.0.0.
```

Comment:
I think we should add the same the text with the explanation of deprecation also here. A user who reads the code, does not automatically look into the documentation.